src/components/product-switcher/style.module.css

```
@@ -101,3 +101,14 @@ Two items to note:
     line-height: 20px;
   }
 }
+
+/*
+TODO: this is temporary for the Vault icon since:
+  - we do not have a yellow version of the icon available from flight-icons yet
+  - the icon is currently on a dark background and appears "invisible"
+*/
+.vaultProductIcon {
+  & path {
+    fill: white;
+  }
+}
```

could we change the fill here to vault yellow? Sorry if that was already discussed!

---

I think we can now grab the token, now available since we copy-pasted in design system tokens. (See https://developer.hashi-mktg.com/swingset/docs/color for demo of `--token-color-vault-brand` rendering as expected):

```suggestion
.vaultProductIcon {
  & path {
    fill: var(--token-color-vault-brand);
  }
}
```

---

I think the yellow is still going through some approval processes? I'm not 100% sure. They specifically asked for white yesterday so that's why I'm making this assumption. I don't think it will hurt to ask about the status of the yellow though!

---

GAH I merged before I saw that you commented @zchsh. My take here is that we _shouldn't_ be setting product icon colors and they should be consumed with the correct colors from flight-icons, so this is a very temporary class and I'm not too concerned with using a token here.